main.go

```diff
@@ -21,13 +21,12 @@ func router(w http.ResponseWriter, r *http.Request) {
 	switch {
 	case strings.Contains(r.Header.Get("Content-type"), "multipart/form-data"):
 		upload(w, r)
-		break
+
```

Comment:
Since this has already been patched before, there's no need of merging this file.